src/buildercore/trop.py

```diff
@@ -180,7 +180,7 @@ def ec2instance(context, node):
     else:
         subnet_id = lu('project.aws.redundant-subnet-id')
 
-    clean_server = _read_script('.clean-server.sh.fragment')
+    clean_server = _read_script('.clean-server.sh.fragment') # this file duplicates scripts/prep-stack.sh
```

Comment:
I have not found a good way of removing this duplication, short of "compiling" bash scripts from a common template. The fragment is inserted into the `UserData` of new EC2 instances so it doesn't have the `#!/bin/bash...set -e...` stuff

Reply:
yeah, me neither, otherwise I would have attempted a fix :|

it's more of a reminder-to-self that there is a design smell there.

Reply:
The full `prep-stack.sh` is only used in AMI creation now, so there may be a way to rename/simplify what it does so that they are not duplicates anymore